content/overview/technology/cluster-managers.md

```diff
@@ -9,8 +9,8 @@ aliases:
   - /intro/technology/cluster-managers
 ---
 
-These tools address different and complementary models to the Platform as a Service workflow we are supporting, like domain microservices and service discovery.
+These tools address different and complementary models to the platform-as-a-service workflow of cloud.gov, like domain microservices and service discovery.
 
-In our research, we have found government agencies typically aren’t in a state where they’re ready for broad and direct deployment of services using those models. We feel the PaaS model is better suited to the most common problems observed in government agencies today.
+Our research suggests government agencies aren’t typically in a state where they’re ready for broad and direct deployment of services using those models. The PaaS model is better suited to the most common problems we observed in government agencies today.
 
-That said, these are powerful tools that we may incorporate in the future, either by exposing them directly, or building services on top of them. We are monitoring the needs we encounter in working on 18F Delivery projects, and will adopt this tech when it will provide a capability we’re in need of to deliver a project not otherwise achievable. Meanwhile we will continue to watch as the community and standards (eg Open Container Initiative) develop.
+That said, these are powerful tools. cloud.gov may incorporate them in the future, either by exposing them directly, or building services on top of them. cloud.gov will adopt this technology when it emerges as the best way to provide a capability that’s necessary to deliver a project. Meanwhile we will continue to watch as the community and standards (for example, the Open Container Initiative) develop.
```

Comment:
Well, rather than "we may incorporate them in future", we **do** incorporate them today. Maybe we can shift this to be about exposing them later when it makes sense?